timeline-chart/src/time-graph-container.ts

```diff
@@ -45,14 +45,19 @@ export class TimeGraphContainer {
             backgroundColor: config.backgroundColor,
             transparent: config.transparent,
             antialias: true,
-            roundPixels: false,
             resolution: ratio,
-            autoResize: true
         });
         this.application.stage.height = config.height;
 
         this.stage = this.application.stage;
         this._canvas = this.application.view;
+        let app: PIXI.Application;
+        app = this.application;
+        canvas.onresize = function (event) {
+            app.renderer.view.width = canvas.width;
+            app.renderer.view.height = canvas.height;
+            app.render();
+        }
```

Comment:
I think if you do }.bind(this) here, then you can use this.application.renderer, etc. instead of passing by a local variable.

The bind method binds the object in parameter to the "this" in the function.

Reply:
removed too!